src/containers/item-details/index.tsx

```diff
@@ -17,12 +21,33 @@ type Props = {
 };
 
 const ProductDetailsArea = ({ space = 1, className, item }: Props) => {
+  const { query } = useRouter();
+  const { itemId } = query;
+  const { ethAddress } = useAppSelector(selectAuthData);
+
   const [selectedImg, setSelectedImg] = useState(item.image.url);
 
+  const [itemOwnerAddress, setItemOwnerAddress] = useState('');
+  const [itemPinataUrl, setItemPinataUrl] = useState('');
+
   const handleImageChange = (img: string) => {
     setSelectedImg(img);
   };
 
+  /* const handleGetItemById = async (id: string) => {
+    const res = await axios.get(`http://localhost:3333/v1/items/${id}`);
+    setItemOwnerAddress(res.data.owner.address);
+    setItemPinataUrl(res.data.image.url);
+  }; */
+
+  useEffect(() => {
+    (async () => {
+      const res = await axios.get(`http://localhost:3333/v1/items/${itemId}`);
```

Comment:
Let's keep the component as much clean as possible. Do not make async calls from it. Use redux actions instead. As Julian commented, please do not hardcode urls. Please have a look at the redux store since the item detail is already making this call. You should be able to pick the owner address from there